src/associated-items.rst

```diff
@@ -48,6 +48,29 @@ An :t:`associated type` shall not be used in the :t:`path expression` of a
 An :t:`associated type` with a :s:`TypeBoundList` shall appear only as an
 :t:`associated trait type`.
 
+:dp:`fls_PeD0DzjK57be`
+A :t:`generic associated type` is an :t:`associated type` with
+:t:`[generic parameter]s`.
+
+:dp:`fls_Y7yIkAbxJpvp`
+A :t:`generic associated type` shall require a :t:`lifetime parameter` when
+the related :t:`implementation` or :t:`trait` has at least one
+:t:`associated function` that imposes a :t:`lifetime bound` on the
+:t:`generic associated type`.
```

Comment:
This is unrelated, associated types can have any kind of generic parameters. The special rules around the where clauses just occur specifically for GATs with lifetime parameters

Reply:
Removed.